sdk/trace/batch_span_processor.go

```diff
@@ -265,7 +266,14 @@ func (bsp *batchSpanProcessor) exportSpans(ctx context.Context) error {
 	}
 
 	if l := len(bsp.batch); l > 0 {
-		global.Debug("exporting spans", "count", len(bsp.batch), "total_dropped", atomic.LoadUint32(&bsp.dropped))
+		global.Debug("exporting spans", "count", len(bsp.batch))
+
+		dropped := atomic.LoadUint32(&bsp.dropped)
+		knownDropped := atomic.SwapUint32(&bsp.knownDropped, dropped)
+		if dropped > knownDropped {
+			global.Warn("dropped spans", "total_dropped", dropped, "known_dropped", knownDropped)
```

Comment:
What value does `known_dropped` represent?

Reply:
known_dropped is the number of spans dropped up until this batch, whereas total_dropped is the number of spans dropped up to **& including** this batch.

In the real world we may see something like: 
(ex: if we only dropped 1 span)
`dropped_spans total_dropped=1, known_dropped=0`

Then if another single span is dropped: 
`dropped_spans total_dropped = 2, known_dropped=1`

It may be useful to have the delta here instead, maybe something like:
`dropped_spans total_dropped=1, dropped_this_batch=1` # `dropped_this_batch=total_dropped - known_dropped`)
`dropped_spans total_dropped=2, dropped_this_batch=1`

Thoughts?

Reply:
I think a delta value is going to be relevant to the users wanting this log message. I think something like `dropped_spans total=2, this_batch=1` makes the most sense.